### PR TITLE
img2pdf: add page

### PR DESCRIPTION
**Pull request by Felipe-noob**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

I hope I did everything right, I had never used GitHub :)

### CODE REVIEW

File: pages/linux/img2pdf.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF
+> More information: <https://pypi.org/project/img2pdf/>.
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:4]: A link to the official repository is better
```suggestion
> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
```

File: pages/linux/img2pdf.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:3]:
```suggestion
> Command-line lossless converter of raster images to PDF.
```

File: pages/linux/img2pdf.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://pypi.org/project/img2pdf/>.
+
+- Convert images to PDF:
+
+`img2pdf {{image1.jpg}} {{image2.jpg}} -o out.pdf`
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:8]: We prefer long arguments when are available:
```suggestion
`img2pdf {{path/to/image1.jpg}} {{path/to/image2.jpg}} --output {{path/to/file.pdf}}`
```

File: pages/linux/img2pdf.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convert all PNGs in the current directory to a single pdf:
+
+`img2pdf *.{{png}} -o out.pdf`
+
+- Convert only the first page of multi-page images to pdf:
+
+`img2pdf example.gif {{--first-frame-only}} -o out.pdf`
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:16]: This is the correct usage of token syntax:
```suggestion
`img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
```

File: pages/linux/img2pdf.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Auto orient to A4 page size in landscape mode and a border of 2cm horizontally and 5.1cm vertically:
+
+`img2pdf example.jpeg --auto-orient --pagesize {{A4^T}} --border {{2cm:5.1cm}} -o out.pdf `
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:20]:
```suggestion
`img2pdf {{path/to/file.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm:5.1cm}} --output {{path/to/file.pdf}}`
```

File: pages/linux/img2pdf.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`img2pdf example.tiff --pagesize {{30cmx20cm}} --imgsize {{10cmx15cm}} --fit {{shrink}} -o out.pdf`
+
+- Set metadata:
+
+`img2pdf example.png --title {{title}} --author {{author}} --creationdate {{1970-01-31}} --keywords {{keywords}} --subject {{subject}} -o out.pdf`
+
+- Force RGB as colorspace:
+
+`img2pdf example.png --colorspace {{RGB}} -o out.pdf
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md]: This is the fix:
```suggestion
`img2pdf {{path/to/file.png}} --colorspace {{RGB}} --output {{path/to/file.pdf}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @Felipe-noob, welcome and thanks for your submission. I left some comments.

Is this command Linux specific? If this works in macOS or Windows should be moved to the `common/` folder.

File: pages/linux/img2pdf.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`img2pdf example.tiff --pagesize {{30cmx20cm}} --imgsize {{10cmx15cm}} --fit {{shrink}} -o out.pdf`
+
+- Set metadata:
+
+`img2pdf example.png --title {{title}} --author {{author}} --creationdate {{1970-01-31}} --keywords {{keywords}} --subject {{subject}} -o out.pdf`
+
+- Force RGB as colorspace:
+
+`img2pdf example.png --colorspace {{RGB}} -o out.pdf`
```
Comment (navarroaxel) [inline pages/linux/img2pdf.md:32]:
```suggestion
`img2pdf {{path/to/file.png}} --colorspace {{RGB}} --output {{path/to/file.pdf}}`
```

File: pages/linux/img2pdf.md (hunk 1 above)
Comment (Felipe-noob) [reply pages/linux/img2pdf.md]: I have commited your edits and will move this page to  `common/`. I think it works outside Linux, but haven't tested anything.

**Comment by navarroaxel** [comment]
@Felipe-noob you don't need to close this PR, you just need to move the file to the `common/` folder.

**Comment by CleanMachine1** [comment]
Just for next time, the `git mv` command would have easy to use. I am unaware if you did use it.

**Comment by marchersimon** [comment]
Or just move it like any other file. In most cases, Git will be aware of that

**Pull request closed without merging**